packages/solid-tiptap/README.md

```diff
@@ -60,16 +60,6 @@ createEffect(() => {
 
 There are out-of-the-box utilities that wraps `createEditorTransaction` based on the [Editor API](https://www.tiptap.dev/api/editor):
 
-- `useEditorCharacterCount`: reactively subscribe to `getCharacterCount`.
-
-```js
-const count = useEditorCharacterCount(() => props.editor);
-
-createEffect(() => {
-  console.log('Character Count:', count());
-});
-```
-
```

Comment:
Removed this because it's not implemented. 

Also, in Tiptap 3 the method to get the character count has been removed, and now it works as a method of an ad-hoc extension.